### PR TITLE
Update reference tests and use npm

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/488551667048375/1203114107513892/f 

### Description
This PR updates all the existing reference tests with the latest versions and adds a simple way to use the privacy reference test repo to update them more frequently.
### Steps to test this PR

Tests should pass

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2362** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2362" target="_blank">Graphite</a> 
    * **PR #2367** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2367" target="_blank">Graphite</a>  👈
      * **PR #2368** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2368" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2367?utm_source=stack-comment).

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
       "dependencies": {
         "@duckduckgo/autoconsent": "github:duckduckgo/autoconsent#v3.0.0",
         "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#5.1.0",
-        "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0"
+        "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0",
+        "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#main"
```
Comment (joshliebe) [inline package-lock.json:14]: Should this be updated to a version? I see [1.0.0](https://github.com/duckduckgo/privacy-reference-tests/releases/tag/1.0.0) was just released

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#5.1.0",
     "@duckduckgo/autoconsent": "github:duckduckgo/autoconsent#v3.0.0",
-    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0"
+    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0",
+    "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#main"
```
Comment (joshliebe) [inline package.json:20]: Should this be updated to a version? I see [1.0.0](https://github.com/duckduckgo/privacy-reference-tests/releases/tag/1.0.0) was just released

**Review by joshliebe — APPROVED** [review]
Other than the version, LGTM 👍

File: package.json (hunk 2 above)
Comment (marcosholgado) [reply package.json:20]: Sorted!

**Pull request merged**